simapp/app.go

```diff
@@ -22,7 +22,7 @@ import (
 	"github.com/cosmos/cosmos-sdk/codec/types"
 	"github.com/cosmos/cosmos-sdk/server/api"
 	"github.com/cosmos/cosmos-sdk/server/config"
-	simappparams "github.com/cosmos/cosmos-sdk/simapp/params"
+	simapparams "github.com/cosmos/cosmos-sdk/simapp/params"
```

Comment:
nit- Is this intended to not actually spenn out "simapp params" anymore?

Personally i think leaving as is `simappparams` is more readable, even with the 3 consecutive 'p's, because "simapp/params" is what it refers to.

Reply:
`simapparams` (with double `p`) was used in 2 other places, so I went by a majority. Don't have a strict opinion. I can change to triple `p`.